mac_commands.go

```diff
@@ -1,6 +1,3 @@
-//go:generate stringer -type=CID
```

Comment:
What is the reason to remove these lines? I think these are helpful as they document the commands to execute the "stringer" utility. Also from within my editor, when I have the file open I can execute the "generate" command and it will re-generate the `..._string.go` files.

Reply:
Sorry, it's my mistake. I put all stringer generate lines in `Makefile`, But I forgot to commit Makefile changes.